openmetadata-ui/src/main/resources/ui/src/components/common/DBTConfigFormBuilder/DBTConfigForm.interface.ts

```diff
@@ -11,11 +11,7 @@
  *  limitations under the License.
  */
 
-import {
-  DbtConfigSource,
-  GCSCredentialsValues,
-  SCredentials,
-} from '../../../generated/metadataIngestion/databaseServiceMetadataPipeline';
+import { DbtConfigSource, GCSCredentialsValues, SCredentials } from '../../../generated/metadataIngestion/databaseServiceMetadataPipeline';
```

Comment:
Are these changes in the UI expected?

Reply:
oh looks like these were autogenerated by a linter/formatter?